Service/src/request_validator/schemas/util_schema.py

```diff
@@ -0,0 +1,52 @@
+import re
+from validate_email import validate_email
+
+NAME_CHAR_LIMIT = 32
+MIN_BUY_POWER = 1
+MAX_BUY_POWER = 1000000
+
+def validateEmail(emailStr, field, errors):
+   if not validate_email(emailStr):
+      errors.append({
+         'InvalidField': field.name + " is an invalid address"
+      })
+   
+   return errors
+
+
+def validatePassword(passwordStr, field, errors):
+   if not re.match("^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)[a-zA-Z\d]{8,32}$", passwordStr):
+      errors.append({
+         'InvalidField': field.name + " must have 8 characters, 1 uppercase letter " +
+            "1 lowercase letter, and 1 number"
+      })
+   
+   return errors
+
+
+def validateName(nameStr, field, errors):
+   if type(nameStr) != str:
```

Comment:
Does this check for names that are empty strings?

Do we care if a name has invalid characters like numbers (I'm assuming names don't normally have numbers)